app/src/main/java/net/squanchy/service/firebase/FirestoreMappers.kt

```diff
@@ -11,32 +12,31 @@ import net.squanchy.service.firebase.model.schedule.FirestoreSpeaker
 import net.squanchy.service.firebase.model.schedule.FirestoreTrack
 import net.squanchy.speaker.domain.view.Speaker
 import net.squanchy.support.checksum.Checksum
-import net.squanchy.support.lang.optional
 import net.squanchy.venue.domain.view.Venue
 import org.joda.time.DateTimeZone
 import org.joda.time.LocalDateTime
 
-fun FirestorePlace.toPlace(): Place = Place(id = id, name = name, floor = floor.optional())
+fun FirestorePlace.toPlace(): Place = Place(id = id, name = name, floor = Option.fromNullable(floor))
 
 fun FirestoreTrack.toTrack(checksum: Checksum) = Track(
     id = id,
     numericId = checksum.getChecksumOf(id),
     name = name,
-    accentColor = accentColor.optional(),
-    textColor = textColor.optional(),
-    iconUrl = iconUrl.optional()
+    accentColor = Option.fromNullable(accentColor),
+    textColor = Option.fromNullable(textColor),
+    iconUrl = Option.fromNullable(iconUrl)
 )
 
 fun FirestoreSpeaker.toSpeaker(checksum: Checksum) = Speaker(
     numericId = checksum.getChecksumOf(id),
     id = id,
     name = name,
     bio = bio,
-    companyName = companyName.optional(),
-    companyUrl = companyUrl.optional(),
-    personalUrl = personalUrl.optional(),
-    photoUrl = photoUrl.optional(),
-    twitterUsername = twitterUsername.optional()
+    companyName = Option.fromNullable(companyName),
```

Comment:
maybe instead of `Option.fromNullable` using `T?.toOption()` would feel more Kotlin-y

Reply:
Will do

Reply:
6b55f9f38b0f5decc257b1f141f2e0f135bea0e6